activesupport/test/time_travel_test.rb

```diff
@@ -186,4 +186,8 @@ def test_time_helper_freeze_time_with_block
 
     assert_operator expected_time.to_s(:db), :<, Time.now.to_s(:db)
   end
+
+  def test_time_helper_unfreeze_time
+    assert_equal method(:unfreeze_time), method(:travel_back)
```

Comment:
I believe this is good, and better than duplicating the suite of `travel_back`. A purist would argue that we are testing the implementation, but the trade-off is great in my view and it trivially guarantees that the suite of `travel_back` passes for `unfreeze_time`.

The only detail is that the the arguments would conventionally go the other way around, because in minitest [the expected value comes first](http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-assert_equal). So, it would be

```ruby
assert_equal method(:travel_back), method(:unfreeze_time)
```

instead.

Reply:
Following syntax conventions are important to avoid potential confusion for someone coming to the test suite at a later date. This has now been changed.

I was initially concerned this might be too simple of a testing approach, but glad you liked the method comparison strategy for testing. I like things like this because if the test suite for `travel_back` changes, then you don't need to worry about keeping tests for `unfreeze` in parallel.